website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -25,6 +25,31 @@ In Nomad 1.3.0, the default raft protocol version has been updated to
 server will automatically upgrade that server's raft protocol. See the
 [Upgrading to Raft Protocol 3] guide.
 
+#### Client State Store
+
+The client state store will be automatically migrated to a new schema
+version when upgrading a client. Downgrading to a previous version of
+the client after upgrading it to Nomad 1.3 is not supported. To
+downgrade safely, users should erase the Nomad client's data
```

Comment:
Would it be good to drain the client first to avoid leaving orphaned task processes? Or maybe restart it after the downgrade? Or does it even matter? 😅

Reply:
On the downgrade case? Yeah, they should definitely drain all tasks. I'll make note of that.

Reply:
Done in [938e8de](https://github.com/hashicorp/nomad/pull/12078/commits/938e8de5ae60627c6b42c5df841797eeb148e69b)